### DISCUSSION TITLE
Add Option To Format Target Device in Display Menu.

### DISCUSSION THREAD

**Discussion by keeratsawhney** [outdoorbits/little-backup-box#342] [Ideas]
I am trying to use LBB as a daily driver and realized the need for formating the target device. Could you please implement this ? @outdoorbits

**Comment by outdoorbits** [comment]
Just have a look at tools in the web UI.

**Reply by keeratsawhney** [reply]
Yes, We have it in tools. But when not using wifi to connect and on the go. I am wishing for a format target device in the display menu over LED Screen that we have. Possible?

**Comment by keeratsawhney** [comment]
![20241221_020740.jpg](https://github.com/user-attachments/assets/3ad0b4ba-c031-4081-859c-55055df85013)

Here in this menu

**Comment by outdoorbits** [comment]
For me it sounds a bit risky to format a drive just by the display menu. The approach should always be a maximum of data protection. Maybe there is another way to meet your needs. Why do you have to format your device that frequent?

**Comment by keeratsawhney** [comment]
I am using external ssd drive to backup the data from SD Card using LBB. I
had a backup of the data from the SSD. But I forgot to format it. While on
the shoot, I realised that there is no space on target i.e SSD. Wanted to
format it so it can have a backup. TBH, it is very inconvenient to connect
mobile. Sometimes we just need basic things right away. I am preparing a
box to have all utilities right from menu without need to access wifi every
time.

**Comment by ruebyi** [comment]
Please don’t add it by default to the Front Page!
It’s too risky - lbb is a perfect tool for quick backups -> and I would like to see the front interface as clean as possible. 

In my opinion this would need some sort of password protection - which would be necessary to be entered by the joystick or the buttons… 
I don’t think that’s convenient

**Reply by outdoorbits** [reply]
I feel exactly the same. I have concerns about making such a dangerous tool available unsecured via the display menu.
In addition, it would be almost impossible to display any error messages in a meaningful way, especially since display output is very volatile. Then there would not only be a risk that a data storage device would be inadvertently formatted, but also that a backup to a damaged file system would be unusable.

**Comment by outdoorbits** [comment]
How is the data transferred from the SSD to a computer? Wouldn't it make sense to move files rather than copy them? This would not be any extra work if the files are copied manually. If you transfer the files to a cloud storage using lbb, you could even use the implemented move function.
